microscopium/screens/cellomics.py

```diff
@@ -29,8 +30,12 @@ def batch_stitch_stack(file_dict, output, order=[0, 1, 2], target_bit_depth=8, *
         files. This dictionary is built using the ``make_key2file`` function.
     output : string
         The directory to output the stitched and concatenated images to.
-    order : list of int, optional
+    stitch_order : array of int, shape (M, N)
+        The order of the stitching.
+        Passed to ``microscopium.cellomics.snail_stitch``
```

Comment:
That's cool! If I want to refer to a function in another module, how do I go about that? ``stretchlim`` is imported from a different module. Do I just refer to ``stretchlim`` or do I still need to write ``preprocessing.stretchlim`` ?